dvc/remote/gdrive.py

```diff
@@ -393,6 +396,23 @@ def _gdrive_download_file(
         ) as pbar:
             gdrive_file.GetContentFile(to_file, callback=pbar.update_to)
 
+    @contextmanager
```

Comment:
just to double check - do we properly pass `close` down to the PyDrive?

Reply:
pydrive2 doesn't expose any `close()` methods anywhere. I think the underlying `httplib2` closes connections before raising errors; so no special handling needed.

Reply:
it should close if just `return` from the block ... we should check the logic carefully here

Reply:
upstream issue: https://github.com/iterative/PyDrive2/issues/44